appscale/tools/remote_helper.py

```diff
@@ -259,6 +284,25 @@ def spawn_nodes_in_cloud(cls, options, count=1):
 
 
   @classmethod
+  def spawn_other_nodes_in_cloud(cls, options, agent, params, count=1):
+    """Starts a single virtual machine in a cloud infrastructure.
```

Comment:
I think the description here should be that it starts 'count' number of virtual machines and not just a single virtual machine.